components/mm-components/modal/MmModal.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div
+    v-if="value"
+    :id="id || randomId"
+    class="mm-modal-backdrop"
+    @click="close"
+  >
+    <div class="mm-modal" tabindex="-1" role="dialog" @click.stop="">
+      <div class="mm-modal-head title-5">
+        <slot name="head"></slot>
+        <button
```

Comment:
Based off the new requirements Paul mentioned in slack (SS), the close button should be able to be hidden. The way you made the component should be able to handle everything else. 

<img width="996" alt="image" src="https://user-images.githubusercontent.com/1762902/203472887-0b19fc63-cdab-4bfe-ab60-c8e14f182be3.png">